### PR TITLE
Bug 1957772: Fix openshift conformance test

### PR DESCRIPTION
**Pull request by oribon**
https://bugzilla.redhat.com/show_bug.cgi?id=1957772
https://bugzilla.redhat.com/show_bug.cgi?id=1960577

### CODE REVIEW

File: bindata/linuxptp/ptp-daemon.yaml [hunk 1]
```diff
@@ -12,6 +12,8 @@ spec:
       app: linuxptp-daemon
   updateStrategy:
     type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 33%
```
Comment (SchSeba) [inline bindata/linuxptp/ptp-daemon.yaml:16]: why you use 33% and not 1 for example any special reason?

Comment (knobunc) [reply bindata/linuxptp/ptp-daemon.yaml:16]: 1 makes upgrades very slow... 10% is what most things do, but components choose something larger.

**Pull request merged**